irods/manager/data_object_manager.py

```diff
@@ -121,23 +121,29 @@ def call___del__if_exists(super_):
         next_finalizer_in_MRO()
 
 
-class ManagedBufferedRandom(io.BufferedRandom):
+def managed_class(cls):
```

Comment:
Is this private to the implementation? i.e. Should users of the library ever reach for this? If not, consider adding a leading underscore to signal that to the reader.